src/src_V2/components/authWrapper/authPageWrapper.jsx

```diff
@@ -0,0 +1,32 @@
+import styles from './authPageWrapper.module.css';
+import logo from '../../assets/Travely.png';
+import loginIllustration from '../../assets/login_illustration.svg';
+import { Typography } from '@mui/material';
+
+const authPageWrapper = (props) => {
```

Comment:
pls use destructuring, and use Mui components to render your UI.

Reply:
Thanks.